src/settings/Settings.tsx

```diff
@@ -649,6 +649,11 @@ export const SETTINGS: { [setting: string]: ISetting } = {
         displayName: _td("settings|always_show_message_timestamps"),
         default: false,
     },
+    "userTimezone": {
+        supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS,
+        displayName: _td("settings|preferences|userTimezone"),
```

Comment:
In `en_EN.json`, the keys is `settings|preferences|user_timezone`.
To be avoid this kind of mistake, you can use `yarn i18n` (https://github.com/matrix-org/matrix-react-sdk/blob/develop/package.json#L36). It will populate the missing translation in `en_EN.json`.